lib/web/databases.go

```diff
@@ -135,6 +135,17 @@ func (h *Handler) handleDatabaseCreate(w http.ResponseWriter, r *http.Request, p
 
 // updateDatabaseRequest contains some updatable fields of a database resource.
 type updateDatabaseRequest struct {
+	// DBOverwrite will completely overwrite an existing database
+	// as if it were a create database request versus a partial update
+	// with select few fields as defined below.
+	// DBOverwrite is used when user opts to "overwrite" an existing database
+	// in the web UI. Partial update is used under the hood by the web UI
+	// when it detects some select fields have changed eg: update CA Cert
+	DBOverwrite *createDatabaseRequest `json:"dbOverwrite,omitempty"`
```

Comment:
would it work if `DBOverwrite` is a bool?

Reply:
dont think so, i need the same field as defined in the `createDatabaseRequest` fields

Reply:
aren't those already in `updateDatabaseRequest` below?

Reply:
i see what you mean, i think it's better to leave it as is, just in case `createDatabaseRequest` adds more fields. 

the alternate you suggest means i have to manually fill out the `createDatabaseRequest` struct and remember to update this struct with the newly added fields

Reply:
Just feels weird to me to have one request in another request.

Are we deprecating the partial update part eventually?

Alternatively I think you can also do something like this

```
type updateDatabaseRequest struct {
   createDatabaseRequest
   DBOverwrite bool`json:"dbOverwrite"`
} 
```
(might need to move CACert to `createDatabaseRequest` though).

Or maybe instead of making the update call, the frontend can make the create call with an overwrite flag?

Reply:
how about this: https://github.com/gravitational/teleport/pull/45180/commits/0beb819c7af4f7af85011c19f662016cad8fadd2

i thought the create call with overwrite flag was a great idea